### ISSUE TITLE
import super_gradients  yolo_nas = super_gradients.training.models.get("yolo_nas_l", pretrained_weights="coco").cuda() yolo_nas.predict("https://deci-pretrained-models.s3.amazonaws.com/sample_images/beatles-abbeyroad.jpg").show()

### ISSUE THREAD

**Issue by VladimirLky** [Deci-AI/super-gradients#947]

**Comment by shaydeci** [comment]
Would be great if you can elaborate (see our issue templates).
Otherwise we sould have to close this issue down, as it is not clear enough.

**Comment by Louis-Dupont** [comment]
@VladimirLky any update ? If not we will close this issue due to lack of information
